packages/backend/src/config/parseConfig.ts

```diff
@@ -925,6 +925,10 @@ export type AuthConfig = {
         allowedOrgRoles: OrganizationMemberRole[];
         maxExpirationTimeInDays: number | undefined;
     };
+    oauthServer?: {
```

Comment:
not sure why this branch is showing changes from another merged PR https://github.com/lightdash/lightdash/pull/15925